apps/ledger-live-desktop/src/renderer/components/WebPlatformPlayer/TopBar.jsx

```diff
@@ -113,33 +115,73 @@ export type Props = {
   onReload: Function,
   onClose?: Function,
   onHelp?: Function,
-  onOpenDevTools: Function,
   config?: TopBarConfig,
+  webview: WebviewTag,
 };
 
 const WebPlatformTopBar = ({
   manifest,
   onReload,
   onHelp,
   onClose,
-  onOpenDevTools,
   config = {},
+  webview,
 }: Props) => {
   const { name, icon } = manifest;
 
   const {
     shouldDisplayName = true,
     shouldDisplayInfo = true,
     shouldDisplayClose = !!onClose,
+    shouldDisplayNavigation = false,
   } = config;
 
+  const [canGoBack, setCanGoBack] = React.useState(false);
+  const [canGoForward, setCanGoForward] = React.useState(false);
   const enablePlatformDevTools = useSelector(enablePlatformDevToolsSelector);
   const dispatch = useDispatch();
 
+  const handleDidNavigate = useCallback(() => {
+    setCanGoBack(webview.canGoBack());
+    setCanGoForward(webview.canGoForward());
+  }, [webview]);
+
+  useEffect(() => {
+    if (webview && shouldDisplayNavigation) {
+      webview.addEventListener("did-navigate", handleDidNavigate);
+      webview.addEventListener("did-navigate-in-page", handleDidNavigate);
```

Comment:
some comment in the code would be valuable. maybe link to a documentation.

what is the different between the 2 events, is it possible they trigger both?